distributions/stacks/openshift/application/openshift-service-mesh/memberroll.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  namespace: istio-system
```

Comment:
would it be possible to include configurable parameters for the namespace ControlPlane and MemberRoll are in?
We provision multiple control planes on one cluster independent of this ODH Operator and ir would be interesting to know if we could reference a custom control plane namespace for open dta hub kubeflow usage.